core/src/utils/helpers.ts

```diff
@@ -9,7 +9,7 @@ export const componentOnReady = (el: any, callback: any) => {
   if (el.componentOnReady) {
     el.componentOnReady().then(callback);
   } else {
-    callback();
+    raf(() => callback(el));
```

Comment:
Hey @liamdebeasi,
Thanks for the fix, it is working well in my app (using 5.7.0-dev.202103121659.630848a).

May I suggest adding a comment and a test ?
Thanks

Reply:
Great glad it's working! Yeah I still need to write tests and add some docs for this. Just wanted to get this code up for some initial testing.